lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -901,6 +901,11 @@ NativeCodeGenerator::CodeGen(PageAllocator * pageAllocator, CodeGenWorkItem* wor
             workItem->GetJITData(),
             scriptContext->GetRemoteScriptAddr(),
             &jitWriteData);
+        if (hr == E_ACCESSDENIED && body->GetScriptContext()->IsClosed())
```

Comment:
There are some cases where we will want to failfast here. For example, in case WriteProcessMemory fails we throw some different hresult and we do want to failfast (as that would be an AV for in-proc JIT).

E_ACCESSDENIED is only thrown in case script context was closed. But maybe we should move to use custom hresults for this sort of thing which are more meaningful for us. At least that makes more clear what code is intended to do.